IDEAS/Buildings/Components/Examples/horizontalFins.mo

```diff
@@ -0,0 +1,63 @@
+within IDEAS.Buildings.Components.Examples;
```

Comment:
I will remove the example, but why remove HFinsAndOverhang model? it is a model combination of HFins and Overhang. Should not be added to the BuildingShade Example along with HorizontalFins?